h2o-automl/src/test/java/ai/h2o/automl/targetencoding/TargetEncodingDistributedTest.java

```diff
@@ -0,0 +1,116 @@
+package ai.h2o.automl.targetencoding;
+
+import org.junit.*;
+import water.MRTask;
+import water.TestUtil;
+import water.fvec.Chunk;
+import water.fvec.Frame;
+import water.fvec.TestFrameBuilder;
+import water.fvec.Vec;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static water.H2O.CLOUD;
+
+public class TargetEncodingDistributedTest extends TestUtil {
+
+  @BeforeClass
+  public static void setup() {
+    stall_till_cloudsize(1);
+  }
+
+  private Frame fr = null;
+
+  /*@Test
```

Comment:
Going to consider to move tests from automl module to ext_te module as a next step